app/metrics.yaml

```diff
@@ -81,7 +81,7 @@ legacy_ids:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/issues/4901
     notification_emails:
-      - jalmeida@mozilla.com
+      - android-probes@mozilla.com
     expires: never
```

Comment:
Same thing here, permanent collections require an individual contributor to be responsible for it.